weaviate/collections/classes/types.py

```diff
@@ -100,3 +135,256 @@ def _check_properties_generic(properties: Optional[Type[Properties]]) -> None:
         and not is_typeddict(properties)
     ):
         raise InvalidDataModelException("properties")
+
+
+ISOCountryCode = Literal[
```

Comment:
do we really need this? 😅

Reply:
Maybe, maybe not. I think it's helpful since users can look it up quickly to see its meaning within the context of Weaviate. It also adds mypy safety if people are using that

Reply:
Hahaha, I didn't see that it got formatted by Black like that. I agree, it looks ridiculous

I will remove it for the pydantic extra types!